arrows/ffmpeg/ffmpeg_video_input.cxx

```diff
@@ -497,6 +498,8 @@ ::open( std::string video_name )
     {
       throw kwiver::vital::video_runtime_exception("Video stream open failed for unknown reasons");
```

Comment:
You can use VITAL_THROW here if the source location of the exception will be useful.

Reply:
That's good to know, but I think outside the scope of this branch.  I could address that on another branch.